jep/200/README.adoc

```diff
@@ -378,29 +385,60 @@ so the maintainers will be the entry point.
+
+== Appendix A. JEP Review Notes
+
+=== Tested Plugins
+
+During JEP-200 review an extra testing has been performed.
+Testing steps and discovered issues are being tracked in
+link: https://docs.google.com/document/d/1uQcyaaLvGFwFDe0mQ27JHeG2icdX0XfCHILbHGOtAmA/edit[JEP-200 Testing Notes].
+
+* link:https://github.com/jenkinsci/acceptance-test-harness[Jenkins Acceptance Test Harness (ATH)] has been executed with the patched components, several plugins were fixed (see the _Testing_ section)
+** Jenkins WAR from link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120] has been tested with a custom core
+** After reviewing of existing ATH tests we concluded that usage of a custom WAR is not a problem
+** We agreed that testing against obsolete dependencies could be a problem.
+During the PR merge procedure the JEP Sponsor and the BDFL Delegate will rerun ATH to confirm there is no issues with stock Jenkins WAR
+* link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compatibility Tester (PCT)]
+** Originally PCT has been executed for a limited plugin set for a sponsor's custom Jenkins WAR.
+** It was decided that it is not enough (not recent plugin versions, potential impact on the plugin behavior by the custom logic),
+so the BDFL delegate and the JEP Sponsor re-run PCT with a standard Jenkins WAR
+** During testing the BDFL Delegate discovered issues which prevent him from running PCT in
+particular cases.
+These issues are listed in the link:https://issues.jenkins-ci.org/browse/JENKINS-48734[JENKINS-48734] EPIC.
+The blocker issues have been resolved.
+** All plugins recommended in the Jenkins Installation Wizard have been covered as well as many other plugins
+** BDFL delegate has not tested Pipeline and Blue Ocean plugins,
+because they are being maintained by employees of the JEP Sponsor's company.
+According to the JEP sponsor, they were covered by their internal testing procedure.
+
+Although there will be extra testing performed before the release of the change in the Weekly,
+BDFL confirms that the current test coverage is good enough to accept this Jenkins Enhancement Proposal
+(Jan 08, 2017).
+
+=== Concern about the single whitelist approach
+
+https://github.com/oleg-nenashev[@oleg-nenashev] raised a concern about using the same whitelist for Remoting and XStream:
+
+* With the reference implementation in link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120]
+there is no way to approve serialization only for a single serialization type
+(e.g. only XStream).
+* Possible attack vectors in Remoting and XStream differxw,
+especially when Remoting CLI is enabled due to the features missing in other CLI modes (multiple file parameters, etc.).
+In such case
+
+Feedback from the JEP Sponsor:
+
+* Remoting CLI is not a concern since we are going to consider it as insecure
+and deprecated option even after integrating
+* Current implementation can be extended in the future if needed.
```

Comment:
But I do not foresee any possible need for it in the future. This section misleadingly implies that _I_ think there is something here which could be improved but which we are deferring.

Reply:
Will make it explicit that there is no plan to work on it

Reply:
(resolved) ?